apps/condo/domains/resident/gql.js

```diff
@@ -72,6 +72,12 @@ const REGISTER_RESIDENT_SERVICE_CONSUMERS_MUTATION = gql`
     }
 `
 
+const SUGGEST_PROVIDER_QUERY = gql`
+    query suggestProviderByTin ($data: SuggestProviderInput!) {
+        result: suggestProvider(data: $data) { tin, name }
```

Comment:
We usually avoid commas in gql types as far as I remember, so...
```suggestion
        result: suggestProvider(data: $data) { tin name }
```